mpyg321/mpyg321.py

```diff
@@ -1,6 +1,22 @@
 import pexpect
 from threading import Thread
 
+
+def is_installed(name):
+    """Checks if a program is installed"""
+    import os
```

Comment:
I do not think that adding imports inside a function is a good practice. I believe this should be at the beginning of the file

Reply:
I have removed this function anyway, so this isn't needed.